hubspot3/engagements.py

```diff
@@ -100,3 +100,31 @@ def get_all_as_generator(self, limit=-1, **options):
 
             yield from engagements
 
+    def get_recently_modified(self, since: int, end_date: int, **options):
+        """get recently modified engagements"""
+        finished = False
+        output = []
+        query_limit = 100  # Max value according to docs
+        offset = 0
+
+        def clean_result(engagement_list, start_d, end_d):
+            output = []
+            v1 = "engagement"
+            v2 = "lastUpdated"
+            for eng in engagement_list:
+                if eng[v1][v2] >= start_d and eng[v1][v2] <= end_d:
+                    output.append(eng)
+            return output
```

Comment:
Imagino que essas linhas 112 e 113 vieram por causa do flake8. Aqui vai uma sugestão de como se adaptar ao limite de colunas:

```suggestion
        def clean_result(engagement_list, start_d, end_d):
            output = []
            for eng in engagement_list:
                eng_update_date = eng["engagement"]["lastUpdated"]
                if eng_update_date >= start_d and eng_update_date <= end_d:
                    output.append(eng)
            return output
```